app/javascript/components/App.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
```

Comment:
- [ ] Kindly install the `react-router-dom` as well. It is missing your `package.json` file.